run.py

```diff
@@ -38,9 +38,8 @@
 RUN_COMMAND = 'sudo ~/env3{py_v}/bin/python serve.py'
 
 AIOH_VERSIONS = {
-    '2.0': 'aiohttp>=2.0,<2.1',
-    '2.2': 'aiohttp>=2.2,<2.3',
-    '2.3a': 'https://github.com/aio-libs/aiohttp/archive/a60bfb4b4b3eb4d530c2871080201d700c6ed0f9.zip',
```

Comment:
Can we use these github links like before to include results for the in-development 3.8 branch and also master (for version 4)?

Reply:
Unfortunately, it's not possible to correctly install pip package from github via archive if package has cython extensions.

See more [here](https://github.com/aio-libs/aiohttp/issues/3907).